.changeset/twenty-items-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-documentation': minor
+'@swisspost/design-system-styles': minor
+---
+
+implement blockquote component
```

Comment:
The blockquote is not a new component, you updated the existing one therefore this is a patch.

```suggestion
---
'@swisspost/design-system-documentation': patch
'@swisspost/design-system-styles': patch
---

Updated the blockquote styles to match the new Post design.
```